__docs__/wonder-blocks-modal/flexible-dialog.stories.tsx

```diff
@@ -0,0 +1,539 @@
+/* eslint-disable no-alert */
```

Comment:
Could we use a `eslint-disable-next-line` comment instead so it is locallzed to the call site where it is intentionally used?

Reply:
I think this was an artifact from `OnePaneDialog` stories. I removed it!